client/src/Components/CreateOffer/index.jsx

```diff
@@ -1,11 +1,179 @@
-import React from 'react';
-import UnderConstruction from '../UnderConstruction';
-
-export default function AutoCompleteTags() {
-  return (
-    <>
-      <h1>AutoCompleteTags</h1>
-      <UnderConstruction />
-    </>
-  );
+import React, { Component } from 'react';
+import { Container, Row, Col, Form, Button } from 'react-bootstrap';
+import { withRouter } from 'react-router-dom';
+
+import AutoCompleteTags from '../CommonComponents/AutoCompleteTags';
+import newOfferValidation from './validationSchema';
+
+import './style.css';
+
+class CreateOffer extends Component {
+  state = {
+    title: '',
+    position: '',
+    description: '',
+    skills: null, // skills id's
+    offerType: null, // offer_types id's
+    // member_id: '',
+    errMsg: '',
+  };
+
+  componentDidMount() {
+    // const member_id = get member_id from local storage
+    // this.setState({ member_id })
+  }
+
+  handleSubmit = () => {
+    // eslint-disable-next-line react/prop-types
+    // const { history } = this.props;
+    const { title, position, description, offerType } = this.state;
+
+    newOfferValidation
+      .validate(
+        { title, position, description, offerType },
+        { abortEarly: false }
+      )
+      .then(() => {
+        this.setState({ errMsg: '' });
+        // send request to the backend with body
+        // axios
+        // .post('/api/v1/offers', { title, position, description, skills, offerType })
```

Comment:
command code. 
just write a short explanation of what will happen

Reply:
oh you mean commented code!! :thinking: 

I think it's ok the commented code here explain itself

Reply:
@ashatat  yeah :sweat_smile:

Reply:
@Fatmasiam I need all of these comments when joining front with back-end